python/hail/docs/tutorials/hail-overview.ipynb

```diff
@@ -0,0 +1,1258 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Overview\n",
+    "\n",
+    "This notebook is designed to provide a broad overview of Hail's functionality, with emphasis on the functionality to manipulate and query a genetic dataset. A genome-wide SNP association test is performed at the end, and demonstrates the need to control for confounding caused by population stratification.\n",
```

Comment:
Feel like "is performed at the end, and demonstrates the need to control..." is awkward. Maybe replace "and" with "which"?